pkg/ingester/ingester.go

```diff
@@ -2846,6 +2846,8 @@ func (i *Ingester) createTSDB(userID string, walReplayConcurrency int) (*userTSD
 		StripeSize:                            i.cfg.BlocksStorageConfig.TSDB.StripeSize,
 		HeadChunksWriteBufferSize:             i.cfg.BlocksStorageConfig.TSDB.HeadChunksWriteBufferSize,
 		HeadChunksEndTimeVariance:             i.cfg.BlocksStorageConfig.TSDB.HeadChunksEndTimeVariance,
+		CollectHeadStatistics:                 i.cfg.BlocksStorageConfig.TSDB.CollectHeadStatistics,
```

Comment:
would it be easier to configure if we use `i.cfg.BlocksStorageConfig.TSDB.IndexLookupPlanningEnabled` instead? Planning without stats is invalid and stats without planning is kind of pointless.

(Also the docs on `IndexLookupPlanningEnabled` mention stats collection too.

i think we can also validate this config in `TSDBConfig.Validate`. Also that the collection time is >0 when stats/planning is enabled. Can you do that?